prep-css-box-model/style.css

```diff
@@ -0,0 +1,10 @@
+body {
+  text-align: center;
+}
+
+button {
+  background-color: red;
+  color: white;
+
+  /* 👇 YOUR CODE HERE 👇 */
+}
```

Comment:
Hi Daniel, try moving your styles in to the style.css file. The buttons should change as the styling is applied.